common/flatpak-dir.c

```diff
@@ -7402,7 +7402,7 @@ flatpak_dir_deploy (FlatpakDir          *self,
                                              checksum,
                                              (char **) subpaths,
                                              installed_size,
-                                             previous_ids);
+                                             (char **)previous_ids);
```

Comment:
Shouldn’t you just make the `previous_ids` argument in the definition of `flatpak_dir_new_deploy_data()` a `const gchar * const *`?

Reply:
yes you're right